third_party/sl4a/README

```diff
@@ -1,4 +1,4 @@
 This folder a heavily modified version of sl4a which acts as an RPC library for
-third-party facades.
+third-party code snippets (facades).
```

Comment:
Yes, this file is describing how our modified sl4a differs from upstream. So I'm trying to use sl4a terminology here and in README.google because I'm relating this to real sl4a.